artistools/data/refspectra/AT2017gfo_ENGRAVE_v1.0_XSHOOTER_MJD-57983.969_Phase+1.43d.dat.meta.yml

```diff
@@ -2,3 +2,6 @@
 dist_mpc: 40
 label: AT2017gfo +1.43d
 t: 1.43
+e_bminusv: 0.11 # smartt 2017
+r_v: 3.1
```

Comment:
Where did you get the spectra from? I can check. They usually aren't

Reply:
Looks like they aren't corrected for reddening (https://sid.erda.dk/wsgi-bin/ls.py?share_id=bJ0ZS1CdFj;current_dir=flux_corrected_smoothed_joined_spectra&flags=f), but I think we should switch to the de-reddened spectra (https://sid.erda.dk/wsgi-bin/ls.py?share_id=bJ0ZS1CdFj;current_dir=dereddened%2Bderedshifted_spectra&flags=f) rather than doing it ourselves.

Reply:
As long as we say what we did it's fine. There isn't much reddening anyway